<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferenceTreeDockWidget</class>
 <widget class="QDockWidget" name="ReferenceTreeDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reference Tree</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnRefTreeAdd">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="litsoz3.qrc">
          <normaloff>:/reftree_addfolder.png</normaloff>:/reftree_addfolder.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRefTreeDelete">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="litsoz3.qrc">
          <normaloff>:/reftree_deletefolder.png</normaloff>:/reftree_deletefolder.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRefTreeDeleteReferences">
        <property name="toolTip">
         <string>remove selected reference from current reference tree folder</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="litsoz3.qrc">
          <normaloff>:/reftree_delete.png</normaloff>:/reftree_delete.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="animated">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="litsoz3.qrc"/>
 </resources>
 <connections/>
</ui>
